### PR TITLE
RT-734-Typescript-decorators-error

### PR DESCRIPTION
**Pull request by yashpmohod**
## Overview
Changed extends the path in the `tsconfig.lib.json` and `tsconfig.json` files.
 
 
## How it was tested
Open charts-on-fhir project in VS code. and run app chart-on-fhir apps locally.

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [ ] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

File: libs/ngx-charts-on-fhir/tsconfig.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "./tsconfig.lib.json"
+  "extends": "../../tsconfig.base.json",
+  "include": ["**/*.ts"]
```
Comment (semicolin) [inline libs/ngx-charts-on-fhir/tsconfig.json:3]: does `tsconfig.lib.prod.json` need to extend `./tsconfig.lib.json` now?

**Review by semicolin — CHANGES_REQUESTED** [review]
Something similar probably needs to be done for each app because I'm still seeing the errors in patient app.component.spec.ts. Also see question below...

File: libs/ngx-charts-on-fhir/tsconfig.json (hunk 1 above)
Comment (yashpmohod) [reply libs/ngx-charts-on-fhir/tsconfig.json:3]: Yes `tsconfig.lib.prod.json` needs to extend `./tsconfig.lib.json` now and also `tsconfig.spec.json` needs to extend `tsconfig.json`.

File: apps/cardiovascular-patient-app/tsconfig.app.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["src/test.ts", "src/polyfills.ts", "src/**/*.spec.ts"],
+  "include": ["**/*.ts"]
```
Comment (semicolin) [inline apps/cardiovascular-patient-app/tsconfig.app.json:4]: I don't think we need to repeat the "include" here because it's in ./tsconfig.json

**Review by semicolin — CHANGES_REQUESTED** [review]
What about the other apps? Don't they need the same changes as patient-app?

File: apps/cardiovascular-patient-app/tsconfig.app.json (hunk 2 above)
Comment (yashpmohod) [reply apps/cardiovascular-patient-app/tsconfig.app.json]: Okay, @semicolin I will remove this.

File: apps/cardiovascular-health/tsconfig.app.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["src/test.ts", "src/polyfills.ts", "src/**/*.spec.ts"]
```
Comment (semicolin) [inline apps/cardiovascular-health/tsconfig.app.json:3]: why are these excluding `src/polyfills.ts`? don't we want polyfills included in the app build?

Comment (yashpmohod) [reply apps/cardiovascular-health/tsconfig.app.json:3]: Okay, @semicolin  I will remove `src/polyfills.ts` from `tsconfig.app.json` for all `charts-on-fhir apps`.

**Review by semicolin — APPROVED** [review]

**Review by amolht — APPROVED** [review]

**Pull request merged**